fastf1/core.py

```diff
@@ -1076,11 +1087,20 @@ def load(self, *, laps=True, telemetry=True, weather=True,
                     logging.warning("Failed to load weather data!")
                     logging.debug("Weather data failure traceback:", exc_info=exc)
 
+            if messages:
+                try:
+                    self._load_race_control_messages(livedata=livedata)
+                except Exception as exc:
+                    logging.warning("Failed to load RC message data!")
```

Comment:
change RC to race control please; imo the abbreviation is not obvious

Reply:
Warning message updated